src/Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -61,15 +61,27 @@ public RpcSettings(NodeSettings nodeSettings)
         /// <param name="nodeSettings">Application configuration.</param>
         private void LoadSettingsFromConfig(NodeSettings nodeSettings)
         {
+            Guard.NotNull(nodeSettings, nameof(nodeSettings));
```

Comment:
this is redundant because nodesettings is passed from the constructor and constructor has the guard

Reply:
Since this is a private method I agree.